src/server/server_config.h

```diff
@@ -1,8 +1,17 @@
 #pragma once
 
-#include <common/network/net_constants.h>
+#include <common/obd_config.h>
 
-struct ServerConfig {
-    int maxConnections = 4;
-    int worldSize = 16;
-};
\ No newline at end of file
+class ServerConfig : public ObdConfig {
+	int m_maxConnections = 4;
+	int m_worldSize = 4;
+	public:
+		ServerConfig(const std::string& filePath);
+		~ServerConfig();
+		
+		int get_maxConnections() const { return m_maxConnections; }
+		void set_maxConnections(const int value) { m_maxConnections = value; }
+
+		int get_worldSize() const { return m_worldSize; }
+		void set_worldSize(const int value) { m_worldSize = value; }
```

Comment:
The getters and setters are not really necessary, they can be public members/ the class can be a struct instead